src/client.rs

```diff
@@ -158,6 +167,7 @@ fn get_cli_args(default_server_key: &str) -> ArgMatches<'_> {
         )
         .subcommand(SubCommand::with_name("list").about("List all disks on a server"))
         .subcommand(SubCommand::with_name("get_jira_tickets").about("get all tickets created"))
+        .subcommand(SubCommand::with_name("setMaintenance").about("Set maintenance status"))
```

Comment:
could you change this to `set_maintenance` so that it follows the format of the other subcommands?